modules/button/react/lib/ButtonStyles.ts

```diff
@@ -453,10 +453,11 @@ function getButtonFocusRing(buttonType: AllButtonTypes): CSSObject {
     case BetaButtonTypes.OutlineInverse:
       return focusRing(2, 2, true, false, buttonColors.focusRingInner, buttonColors.focusRingOuter);
     case TextButtonTypes.Default:
-    case TextButtonTypes.Inverse:
     case TextButtonTypes.AllCaps:
+      return focusRing(2, 0, true, false, buttonColors.focusRingInner, buttonColors.focusRingOuter);
```

Comment:
This can just use the default params so you can combine these cases with the ones above for `focusRing(2, 0)`

Reply:
Changing this to `focusRing(2, 2)`, otherwise the inverse will not get the white border. Is that ok?